cloudtools/scripts/aws_create_instance.py

```diff
@@ -163,7 +165,17 @@ def create_instance(name, config, region, key_name, ssh_key, instance_data,
                                 ssh_key=ssh_key, instance_data=instance_data,
                                 deploypass=deploypass, reboot=reboot)
             break
+        except NetworkError as e:
+            # it takes a while for the machine to start/reboot so the
+            # NetworkError exception is quite common, just log the error,
+            # without the full stack trace
+            log.warn("problem assimilating %s (%s, %s) - %s, retrying in "
```

Comment:
This message should be adjusted -- something like "cannot connect; instance may still be starting"